behaviours/common.go

```diff
@@ -0,0 +1,72 @@
+package behaviours
+
+// Hookable must be implemented by those entities that attend to all DB Hooks
+type Hookable interface {
+	AfterLoader
+	BeforeSaver
+	AfterSaver
+	BeforeUpdater
+	AfterUpdater
+	BeforeInserter
+	AfterInserter
+	BeforeDeleter
+	AfterDeleter
+}
+
+// AfterLoader must be implemented by those entities listening the AfterLoad event
+type AfterLoader interface {
```

Comment:
i don't know if we are going to need these interfaces at all. The code will be generated and we will not check if a record fulfills them, so the only use is if someone maybe wants to check it by hand